### PR TITLE
Improved styling for community results cta

### PR DESCRIPTION
**Pull request by Lukewh**

### CODE REVIEW

**Comment by Lukewh** [comment]
## QA

- Search for Kibana, the "Show x community results" should be centred.

File: jujugui/static/gui/src/app/components/search-results/search-results.js [hunk 1]
```diff
@@ -515,14 +515,15 @@ YUI.add('search-results', function(Y) {
           'hidden': !this.state.showCommunity && hasPromulgated
         });
       const buttonTitle = this.state.showCommunity ?
-        'Hide community results' : 'Show community results';
+        'Hide community results' : `Show ${community.length} community results`;
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/search-results/search-results.js]: Nice improvement.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7292/

**Comment by Lukewh** [comment]
:shipit: thanks @frankban

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Comment by frankban** [comment]
Let's try again/
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**